customcommands/assets/customcommands-editcmd.html

```diff
@@ -6,6 +6,12 @@ <h2>Custom commands</h2>
 
 {{template "cp_alerts" .}}
 
+<style>
+    .cc-editor {
+        font-family: Consolas;
```

Comment:
```suggestion
        font-family: Consolas, monospace;
```
As [w3schools](https://www.w3schools.com/cssref/pr_font_font-family.asp) highly recommends:
> Start with the font you want, and always end with a generic family, to let the browser pick a similar font in the generic family, if no other fonts are available.